src/unity/python/turicreate/visualization/vg2png

```diff
@@ -47,5 +48,5 @@ function render(spec) {
     .initialize()
     .toCanvas()
     .then(function(canvas) { writePNG(canvas, outputFile); })
-    .catch(function(err) { console.error(err); });
-}
\ No newline at end of file
+    .catch(function(err) { process.exit(_CANVAS_PREBUILT_NON_EXISTENT); });
```

Comment:
Let's keep the `console.error` (before the process.exit). Ideally the user should be able to see the whole error message, in case it's something weird and isn't the missing canvas-prebuilt. (This will catch all possible errors, so it's possible it won't be because of that.)